independent-publisher-connectors/Forem/apiDefinition.swagger.json

```diff
@@ -0,0 +1,980 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0",
+    "title": "Forem",
+    "description": "Access Forem articles, users and other resources via API.",
```

Comment:
@Laskewitz please remove the word "API" from the description. please follow our requirements [here ](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#connector-info)and [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/certification-submission#write-a-description-for-your-connector).